Sources/StreamChatUI/ChatChannelList/ChatChannelListVC_Tests.swift

```diff
@@ -30,6 +30,8 @@ class ChatChannelListVC_Tests: XCTestCase {
     override func setUp() {
         super.setUp()
 
+        UIView.setAnimationsEnabled(false)
```

Comment:
if this is set to `false` in this tests setup, it should be set to `true` when this tests are ended

Reply:
✅